### PR TITLE
stacktrace translation for .net - awsxrayexporter

### PR DESCRIPTION
**Pull request by bhautikpip**
**Description:** <Describe what has changed.>
Added support for stack trace translation for .NET language - awsxrayexporter

**Link to tracking Issue:** <Issue number if applicable>

**Testing:** <Describe what testing was performed and which tests were added.>
Added unit tests for testing

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

File: exporter/awsxrayexporter/translator/cause.go [hunk 1]
```diff
@@ -405,6 +407,73 @@ func fillJavaScriptStacktrace(stacktrace string, exceptions []awsxray.Exception)
...
+						Path:  aws.String(path),
+						Label: aws.String(label),
+						Line:  aws.Int(lineNumber),
+					}
+
+					exception.Stack = append(exception.Stack, stack)
+				}
+			}
+		}
+		// We skip "..." (common frames) and Suppressed By exceptions.
```
Comment (anuraaga) [inline exporter/awsxrayexporter/translator/cause.go:468]: Does dotnet have these or accidental copy paste?

**Review by anuraaga — APPROVED** [review]

File: exporter/awsxrayexporter/translator/cause.go (hunk 1 above)
Comment (bhautikpip) [reply exporter/awsxrayexporter/translator/cause.go]: accidental copy paste. removed it.

**Comment by bhautikpip** [comment]
@kbrockhoff can you also take a look?

**Review by mxiamxia — APPROVED** [review]
Thanks!

**Pull request merged**